lib/index.ts

```diff
@@ -99,8 +104,8 @@ const sdk = (...args: Parameters<typeof createSdk>) => {
       }
       if (refreshToken) {
         try {
-          await this.validateToken(refreshToken)
-          return (await this.refresh(refreshToken)).data
+          await sdk.validateJwt(refreshToken)
+          return (await sdk.refresh(refreshToken)).data as any // the types here don't actually match
```

Comment:
Can we return the result of `validateJwt` to make it consistent and have the correct type?